content/docs/command-reference/commit.md

```diff
@@ -140,83 +138,83 @@ stage with `dvc run --no-commit`, or reproduce an entire pipeline using
 development of the stage is finished, `dvc commit` can be used to store data
 files in the cache.
 
-In the `featurize.dvc` stage, `src/featurize.py` is executed. A useful change to
-make is adjusting a parameter to `CountVectorizer` in that script. Namely,
-adjusting the `max_features` value in the line below changes the resulting
-model:
+In the `featurize` stage, `src/featurization.py` is executed. A useful change to
+make is adjusting the `max_features` parameter to `CountVectorizer` in that
+script. The parameters are defined in `params.yaml` file. Updating the value of
+`max_features` to 6000 in `params.yaml` changes the resulting model:
```

Comment:
Nice but:

- I guess no need to even mention CountVectorizer at this point. Just "parameter for that script."
- "defined in `params.yaml` file" has a basic grammar issue pervasive in your writing.
- No need to mention `max_features` twice, it can probably be written so it's only needed once?